subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/JavaExecToolchainIntegrationTest.groovy

```diff
@@ -25,6 +25,12 @@ import spock.lang.Unroll
 
 class JavaExecToolchainIntegrationTest extends AbstractPluginIntegrationTest {
 
+    def setup() {
+        executer
```

Comment:
Better to be

```
executer.beforeExecute {
    withArgument(xxx)
    ...
}
```

Because the arguments are cleared after each invocation. Not putting them inside `beforeExecute` may cause weird failures that are hard to find.